Thanakumares/07-10-2018/Matrix.js

```diff
@@ -0,0 +1,12 @@
+var arr=[[1,1,1],[1,0,1],[1,1,1]];
+var output="Welcome";
+for (let i = 0; i < arr.length; i++) {
+    const element=arr[i];
+    for (let j = 0; j < element.length; j++) {
+        if(element[j]*1==0){
+            output= `Row ${i} and Column ${j}`;
```

Comment:
once it is found .. u should break and exit the loop as i told only one zero is there in multi-array